ajax/libs/angular-animate/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "angular-animate",
+  "filename": "angular-animate.min.js",
+  "description": "AngularJS module for CSS-based animations (keyframes and transitions) as well as JavaScript-based animations.",
+  "version": "1.7.9",
```

Comment:
```suggestion
```

Please remove the `version` attr, the auto-update bot will populate this itself once it pulls in versioned files for the library.